src/modules/grubcfg/main.py

```diff
@@ -33,11 +33,11 @@ def modify_grub_default(partitions, root_mount_point, distributor):
     default_dir = os.path.join(root_mount_point, "etc/default")
     default_grub = os.path.join(default_dir, "grub")
     distributor_replace = distributor.replace("'", "'\\''")
-    plymouth_bin = os.path.join(root_mount_point, "usr/bin/plymouth")
+    plymouth_bin = libcalamares.utils.chroot_call("sh -c \"which plymouth\"")
     use_splash = ""
     swap_uuid = ""
 
-    if os.path.exists(plymouth_bin):
+    if plymouth_bin == 0:
```

Comment:
In Python, empty strings are falsy, so I think it would be more Pythonic to just do `if plymouth_bin:`